SparkPost/2web2ui
# AC-1973: Global tracking for all forms in webhooks, join, onboarding and passwordReset

AC-1973: Global tracking for all forms in webhooks, join, onboarding and passwordReset

### What Changed

- Allowlisted forms for webhooks, join, onboarding and passwordReset
- Blocklisted the sensitive fields from these form
- Added cypress tests for these forms

### How To Test

- Check for form submissions on webhooks and check if all the sensitive fields are blocklisted
- Check for form submissions on join and onboarding ChoosePlan page.
- Check for form submissions on password reset page 

### To Do

- [ ] Removed a few unit tests from PlanPicker (they didn't seem very useful), may be they should be replaced with some new tests.
